cvpysdk/backupset.py

```diff
@@ -663,6 +696,278 @@ def _update(self, backupset_name, backupset_description, default_backupset):
 
         return self._process_update_reponse(request_json)
 
+    @staticmethod
+    def _get_timestamp(timestamp):
+        """ Returns the timestamp which are in the format %Y-%m-%d %H:%M:%S

```

Comment:
Please correct the Doc-string here.
The function is expecting the time in the format %Y-%m-%d %H:%M:%S, here,
and returns the Epoch time